packages/gluestick/src/commands/compareVersions/compareModuleVersions.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+const semver = require('semver');
+const chalk = require('chalk');
+const path = require('path');
+
+const packageName = '/package.json';
+
+/**
+ * Opens the project's package.json file and compares version requirements to
+ * the actual version of each dependency and devDependency. It lists the
+ * node_modules that don't satisfy the requirement or are missing and suggests
+ * running `npm install` to amend the issue.
+ *
+ * Corner cases: there are no dependencies and/or devDependencies, the version
+ * is a path to a file, the input file to the package.json or node_modules dir
+ * is either undefined or incorrect.
+ *
+ * Return - returns array of dependencies that are missing or out of date.
+ */
+const compareModuleVersions = (projectPackage, modulePath, logger) => {
+  const discrepancies = [];
+  let devDepDifference = false;
+  let allDependencies = projectPackage;
+
+  if (!projectPackage.dependencies || !projectPackage.devDependencies) {
+    if (!projectPackage.dependencies || !projectPackage.devDependencies) {
+      return discrepancies;
+    }
+    allDependencies = projectPackage.dependencies || projectPackage.devDependencies;
+  } else {
+    const totalNumDependencies = Object.keys(projectPackage.dependencies).length +
+      Object.keys(projectPackage.devDependencies).length;
+    allDependencies = { ...projectPackage.dependencies, ...projectPackage.devDependencies };
+    if (Object.keys(allDependencies).length < totalNumDependencies) {
+      devDepDifference = true;
+    }
+  }
+
+  Object.keys(allDependencies).forEach(dep => {
+    // const tempPath = modulePath.concat('/', dep, packageName);
+    const tempPath = path.join(modulePath, dep, packageName);
+    let tempPackage;
+    try {
+      tempPackage = require(tempPath);
+    } catch (e) {
+      logger.error(`${chalk.red('The node_module ')}`, `${chalk.yellow(dep)}`, `${chalk.red(' does not exist')}`);
```

Comment:
Please remove `chalk.red` from this template string (you can leave `yellow` if you want). `logger.error` will mark the string as a error and we want to maintain consistency between logging various stuff.
So it can look sth like this: `The node_module ${chalk.yellow(dep)} does not exist`.